drivers/windows/file_access_windows.cpp

```diff
@@ -77,7 +77,7 @@ Error FileAccessWindows::_open(const String &p_path, int p_mode_flags) {
 		return ERR_INVALID_PARAMETER;
 	}
 
-	/* pretty much every implementation that uses fopen as primary
+	/* Pretty much every implementation that uses fopen as primary
```

Comment:
If fixing comment style, it should be fixed fully ;) (end with a dot)

https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html#comment-style-guide